### DISCUSSION TITLE
Issue with managing users

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#3000] [General Questions & Discussion] [answered]
If you go to /admin and go to accounts, try to delete an existing account (logged in as admin). I do this by selecting the row, then hitting the button DELETE on the right. After 4 seconds, the page containing the accounts jumps to this url (https://ourworld.camera/signin?sign_in&sign_in_destination=admin&sign_in_reason=admin_no_permission) and I get a sign into hubs screen. When returning to the accounts page to check whats happened, nothing. The account I want to delete is there. Seems there is no way for me to delete an account.

**Comment by robinkwilson** [comment, marked as answer]
Good question. Answered here: https://github.com/mozilla/hubs/discussions/2747
